core/src/main/groovy/noe/tomcat/configure/ConnectorTomcatAbstract.groovy

```diff
@@ -28,6 +28,8 @@ abstract public class ConnectorTomcatAbstract<E extends ConnectorTomcatAbstract>
   private String address
   private Integer connectionTimeout
   private Integer redirectPort
+  private UpgradeProtocolTomcat upgradeProtocol
```

Comment:
UpgradeProtocal element is only HTTP connector related, this code is shared with AJP connector as well.